File: 3. Stack/Stack Implementation With Fixed Array/fixedStack.py
stack = [];
stackPointer = -1;
stackLength = 10;

print("Hello! Here You Can Perform Following tack Operation!");
while (True) :
    print("1. Push(Insert)\n2. Pop(Delete)\n3. Peek(See Last Element)\n4. Is Empty(Checks weather Stack is Empty)\n5. Is Full(Checks weather Stack is Full)\n6. Exit");
    choice = int(input("Enter Your Choice:- "));
    if(choice == 1):
        if (stackPointer + 1 == stackLength):
            print("\n~~ Stack is Full ~~\n");
            continue;
        stackPointer += 1;
        val = int(input("\nEnter Your Number:-"));
        stack.append(val);
        print("Insertion Successfull:- ", val, "\n");
        continue;
    elif(choice == 2):
        if (stackPointer == -1):
            print("\n~~ Stack is Empty ~~\n");
            continue;
        print("\nDeletion Successfull:- ", stack[stackPointer], "\n");
        stackPointer -= 1;
        continue;
    elif(choice == 3):
        if (stackPointer == -1):
            print("\n~~ Stack is Empty ~~\n");
        else: 
            print("\nData Element:- ", stack[stackPointer], "\n");
        continue;
    elif(choice == 4):
        if (stackPointer == -1):
            print("\n~~ Stack is Empty ~~\n");
        else: 
            print("\n~~ Stack is Not Empty ~~\n");
        continue;
    elif(choice == 5):
        if (stackPointer + 1 == stackLength):
            print("\n~~ Stack is Full ~~\n");
        else: 
            print("\n~~ Stack is Not Full ~~\n");
        continue;
    elif(choice == 6):
        print("\n~~ Thanks For Using! Have a Great Day! ~~\n");
        break;
    else:
        print("\n~~ Please Enter a Vaild Input! ~~\n");
        continue;

File: 3. Stack/Stack Implementation With Fixed Array/fixedStack.js
const prompt = require('prompt-sync')({ sigint: true });    

(() => {
    let stack = [], stackPointer = -1, stackSize = 5, choice;
    console.log("Hello! Here You Can Perform Following Stack Operation!");
    while (1) {
        console.log("1. Push(Insert)\n2. Pop(Delete)\n3. Peek(See Last Element)\n4. Is Empty(Checks weather Stack is Empty)\n5. Is Full(Checks weather Stack is Full)\n6. Exit");
        choice = Number.parseInt(prompt("Enter Your Choice:- "));
        switch (choice) {
            case 1:
                if (stackPointer + 1 == stackSize) {
                    console.log(`\n~~ Stack is Full ~~\n`);
                    break;
                }
                console.log("\n");
                var num = Number.parseInt(prompt("Enter Your Number:- "));
                stackPointer++;
                stack[stackPointer] = num;
                console.log(`Insertion Successfull:- ${num}\n`);
                break;
            case 2:
                if (stackPointer == -1) {
                    console.log(`\n~~ Stack is Empty ~~\n`);
                    break;
                }
                console.log(`\nDeletion Successfull:- ${stack[stackPointer]}\n`);
                stackPointer--;
                break;
            case 3:
                if (stackPointer == -1) {
                    console.log(`\n~~ Stack is Empty ~~\n`);
                } else {
                    console.log(`\nData Element:- ${stack[stackPointer]}\n`);
                }
                break;
            case 4:
                if (stackPointer == -1) {
                    console.log(`\n~~ Stack is Empty ~~\n`);
                } else {
                    console.log(`\n~~ Stack is Not Empty ~~\n`);
                }
                break;
            case 5:
                if (stackPointer + 1 == stackSize) {
                    console.log(`\n~~ Stack is Full ~~\n`);
                } else {
                    console.log(`\n~~ Stack is Not Full ~~\n`);
                }
                break;
            case 6:
                console.log(`\n~~ Thanks For Using! Have a Great Day! ~~\n`);
                return;
            default:
                console.log(`\n~~ Please Enter a Vaild Input! ~~\n`);
                break;
        }
    }
    return;
})();